RSDKv5/RSDK/Storage/Storage.cpp

```diff
@@ -6,6 +6,10 @@ using namespace RSDK;
 #include "Legacy/UserStorageLegacy.cpp"
 #endif
 
+#if RETRO_PLATFORM == RETRO_KALLISTIOS
+#include <kos.h>
```

Comment:
This ifdef and include might be unneded since kos is included in `RetroEngine.hpp`

Reply:
Actually... looks like it isn't...

Reply:
OH WAIT I SEE